<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModificatorElementWidget</class>
 <widget class="QWidget" name="ModificatorElementWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="CTiglPointWidget" name="center" native="true"/>
   </item>
   <item>
    <widget class="CTiglPointWidget" name="normal" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="dimensionWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Height</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="heightSpinBox">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="maximum">
         <double>999999999999999993220948674361627976461708441944064.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Width</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="widthSpinBox">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="maximum">
         <double>999999999999999993220948674361627976461708441944064.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Area</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="areaSpinBox">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="maximum">
         <double>999999999999999993220948674361627976461708441944064.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CTiglPointWidget</class>
   <extends>QWidget</extends>
   <header>modificators/CTiglPointWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
